loops.js

```diff
@@ -6,12 +6,22 @@
 // ***
 // ****
 // *****
+const VerwachteUitkomsten = ["*", "**", "***", "****", "*****"];
 
+for (let i = 0; i < VerwachteUitkomsten.length; i++) {
+    console.log(VerwachteUitkomsten[i])
```

Comment:
Interessant oplossing! Maar niet ideaal, want stel je wilt 2000 sterretjes? Ga je die dan allemaal handmatig in de array plaatsen?

Bestudeer nog even goed de repeat methode. Die maakt dit namelijk een stuk eenvoudiger en dynamischer.